src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -447,11 +471,30 @@ class ExplorerButtonGroup extends React.Component {
       return this.state.manifestEntryCount > 0;
     }
     if (buttonConfig.type === 'export-to-pfb') {
-      return !this.state.exportingToCloud;
+      // disable the pfb export button if any other pfb export jobs are running
+      return !(this.state.exportingToTerra || this.state.exportingToSevenBridges);
```

Comment:
shoudl we also check for `isPFBRunning` here? too many button types 🤯

Reply:
Haha yeah, I find this code hard to follow. I'll be honest that I don't fully understand it, but the original code didn't check for `isPFBRunning` and as far as I can tell the `isPending` check for export-to-pfb uses `isPFBRunning() == true` to decide if the button is pending (https://github.com/uc-cdis/data-portal/pull/686/files/67fe2b01dd812ce6a929815eebb95f403beaf83d#diff-a6734b941e5a6867b94a16ee82e40e96L471). So I thought it might have been intentional, so that the button is pending but not disabled when isPFBRunning() == true and we're not exporting to Terra (or seven bridges)

Reply:
yeah it was not in there in the first place, i'm ok with that 👍